blueprints/component-test/index.js

```diff
@@ -6,6 +6,7 @@ const isPackageMissing = require('ember-cli-is-package-missing');
 const getPathOption = require('ember-cli-get-component-path-option');
 
 const useTestFrameworkDetector = require('../test-framework-detector');
+const { isModuleUnificationProject } = require('../module-unification');
```

Comment:
This will break on node@4

Reply:
Thanks @ro0gr, I'll update now

Reply:
https://github.com/emberjs/ember.js/pull/16261